code/modules/mob/living/carbon/human/species.dm

```diff
@@ -975,12 +975,12 @@ GLOBAL_VAR_INIT(join_as_robot_allowed, TRUE)
 /datum/species/robot/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
 	. = ..()
 	H.speech_span = SPAN_ROBOT
-	H.health_threshold_crit = -100
+	H.health_threshold_crit -= 50
 
 /datum/species/robot/post_species_loss(mob/living/carbon/human/H)
 	. = ..()
 	H.speech_span = initial(H.speech_span)
-	H.health_threshold_crit = initial(H.health_threshold_crit)
+	H.health_threshold_crit += 50
```

Comment:
Bad idea.

What if we add a species with crit threshold 0 ?

Reply:
I suggested this over direct setting on gain/loss because it'd play nicer with anything else that wants to adjust crit theshold in future. I'm not sure how a threshold 0 species would cause issues with this?